k8s/sonarqube/schema.yaml

```diff
@@ -0,0 +1,51 @@
+application_api_version: v1beta1
+properties:
+  name:
+    type: string
+    x-google-marketplace:
+      type: NAME
+  namespace:
+    type: string
+    x-google-marketplace:
+      type: NAMESPACE
+  sonarqube.image:
+    type: string
+    default: $REGISTRY:$TAG
+    x-google-marketplace:
+      type: IMAGE
+  postgresql.db.password:
+    type: string
+    x-google-marketplace:
+      type: GENERATED_PASSWORD
+      generatedPassword:
+        length: 12
+  postgresql.image:
+    type: string
+    default: $REGISTRY/postgresql9:$TAG
+    x-google-marketplace:
+      type: IMAGE
+  metrics.image:
+    type: string
+    default: $REGISTRY/prometheus-to-sd:$TAG
+    x-google-marketplace:
+      type: IMAGE
+  metrics.enabled:
+    type: boolean
+    title: Enable Stackdriver Metrics Exporter
+    description: Your GCP project should have Stackdriver enabled. For non-GCP clusters, export of metrics to Stackdriver is not supported yet
+    default: false
+  postgresql.exporter.image:
+    type: string
+    default: $REGISTRY/postgresql9/exporter:$TAG
+    x-google-marketplace:
+      type: IMAGE
+
+required:
+- name
+- namespace
+- sonarqube.image
+- postgresql.db.password
+- postgresql.image
+- metrics.image
+- metrics.enabled
+- postgresql.exporter.image
```

Comment:
Please sort this file  - postgresql.exporter should be near postgresql.image

Reply:
sorted